scripts/new_bare_repo

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+usage()
+{
+cat << EOF
+usage: new_bare_repo <hutch> <repo name>
+
+Script for automating the tedious process of integrating an existing IOC into our git bare repo scheme
+Run in the top level of your IOC
+
+Ex. new_bare_repo rix new_device
+(Creates a new bare repo in /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/rix/ named "new_device.git")
+
+You also might need to run kinit and aklog before hand!
+
+EOF
+}
+
+if [ $# -lt 1 ]; then
+   echo 'Need to specify the hutch and name of the proposed bare repo' >&2
+   usage
+   exit 1
+
+elif [[ ($1 == "--help") || ($1 == "-h") ]]; then
+   usage
+   exit 0
+fi
+
+HUTCH=$1
+REPO=$2
+
+if [ -d "/afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/" ] && [ ! -d "/afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/${REPO}.git" ]
+then
+    git init
```

Comment:
Out of scope (?): a future version of this script could double-check that we're not running `git init` in a random directory (some sort of path matching against ${REPO}?)

Reply:
Hmm... The issue is that peoples dev directories, where they are likely building these IOCs, are unlikely to follow the same kind of path structure. It would be easy to add something that looks for ${REPO} _somewhere_ in the path, but that wouldn't cover all possible cases

Reply:
It needs some thought for sure, but I think it would be awkward to accidentally/automatically run git init somewhere in the middle of the file tree